hydro_deploy/core/src/hydroflow_crate/build.rs

```diff
@@ -130,7 +147,11 @@ pub async fn build_crate(
                                     let path_buf: PathBuf = path.clone().into();
                                     let path = path.into_string();
                                     let data = std::fs::read(path).unwrap();
```

Comment:
seems kinda weird to need to load the binary into memory and cache that?

Reply:
I think when I originally wrote this the worry was that another compilation task might overwrite the binary at this path? But I may have been unnecessarily worried...